HyperVExtension/src/HyperVExtension/Models/ProgressObject.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HyperVExtension.Models;
+
+/// <summary>
+/// Represents progress of an operation that require transferring bytes from one place to another.
+/// </summary>
+public class ProgressObject
```

Comment:
`ProcessObject` too generic.  Maybe change the name to `ByteTransferProgress`.  With a descriptive class name, the comment isn't needed.

Reply:
updated, thanks